guides/common/modules/proc_host_registration_steps.adoc

```diff
@@ -0,0 +1,44 @@
+.Procedure
+
+. Generate registration command
++
+UI: In the {ProjectWebUI}, navigate to *Hosts > Register Host* and generate the registration command.
++
+Or you can generate command via API:
+ifdef::foreman-el,foreman-deb[]
++
+[options="nowrap" subs="+quotes,attributes"]
+----
+# hammer host-registration generate-command
+
+# or manually:
+
+# curl -X POST https://{foreman-example-com}/api/registration_commands \
+    --user username:password \
+    -H 'Content-Type: application/json'
+----
+endif::[]
+ifdef::katello,satellite[]
++
+[options="nowrap" subs="+quotes,attributes"]
+----
+# hammer host-registration generate-command --activation-keys your-key
+
+# or manually:
+
+# curl -X POST https://{foreman-example-com}/api/registration_commands \
+    --user username \
```

Comment:
Actually it is only removed here. Can you either keep or remove for both? To keep consistency.

Reply:
If you remove it, I suggest to add a sentence below telling the user what is the format: `username:password`.